### PR TITLE
Conform to new sklearn api

### PR DESCRIPTION
**Pull request by icfly2**
Made import compliant with newer API as per warning:

```
  File "/app/model/model.py", line 3, in <module>
    import eli5
  File "/usr/local/lib/python3.7/site-packages/eli5/__init__.py", line 13, in <module>
    from .sklearn import explain_weights_sklearn, explain_prediction_sklearn
  File "/usr/local/lib/python3.7/site-packages/eli5/sklearn/__init__.py", line 21, in <module>
    from . import transform as _
  File "/usr/local/lib/python3.7/site-packages/eli5/sklearn/transform.py", line 6, in <module>
    from sklearn.feature_selection.base import SelectorMixin  # type: ignore
  File "/usr/local/lib/python3.7/site-packages/sklearn/feature_selection/base.py", line 12, in <module>
    _raise_dep_warning_if_not_pytest(deprecated_path, correct_import_path)
  File "/usr/local/lib/python3.7/site-packages/sklearn/utils/deprecation.py", line 143, in _raise_dep_warning_if_not_pytest
    warnings.warn(message, FutureWarning)
FutureWarning: The sklearn.feature_selection.base module is  deprecated in version 0.22 and will be removed in version 0.24. The corresponding classes / functions should instead be imported from sklearn.feature_selection. Anything that cannot be imported from sklearn.feature_selection is now part of the private API.
```
Yes this breaks backward compatibility for sklearn but will allow eli5 to continue to work with the next version of sklearn 0.24

### CODE REVIEW

**Comment by jimbudarz** [comment]
As with #397, changing the requirements and setup to require sklearn >= 0.20 would maintain compatibility.

**Comment by icfly2** [comment]
sklearn version bump in requirements.txt and setup.py in #397

**Comment by lopuhin** [comment]
Thanks you! This were merged in eli5-org/eli5#2 and released to PyPI with v0.11
